plugins/Jobs.cpp

```diff
@@ -265,7 +265,8 @@ bool BedrockPlugin_Jobs::processCommand(BedrockNode* node, SQLite& db, BedrockNo
             SQResult result;
             if (!db.read("SELECT 1 "
                          "FROM jobs "
-                         "WHERE jobID = " + safeParentJobID + ";",
+                         "WHERE jobID = " +
+                             safeParentJobID + ";",
```

Comment:
I'm confused. The difference in this file is only whitespace, correct? This is presumably from running this code through the auto-formatter (whichI did). I may not have even needed to do this, I don't see any other changes in this file, so I'm not sure how it got caught by the formatter to make any changes.

In your original PR the only difference was to change BedrockNode::Plugin to BedrockPlugin, but that was done separately in a different recent PR.